backend/src/paymaster/index.ts

```
@@ -178,6 +180,80 @@ export class Paymaster {
+        returnValue = {
+          paymaster: paymasterAddress,
+          paymasterData: paymasterData,
+        }
+      }
+
+      return returnValue;
+    } catch (err: any) {
+      if (err.message.includes("Quota exceeded"))
+        throw new Error('Failed to process request to bundler since request Quota exceeded for the current apiKey')
+      if (log) log.error(err, 'signV08');
+      throw new Error('Failed to process request to bundler. Please contact support team RawErrorMsg:' + err.message)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)